tests/subsys/dfu/dfu_target_mcuboot/testcase.yaml

```diff
@@ -1,5 +1,6 @@
 tests:
   dfu.dfu_target_mcuboot:
     tags: dfu mcuboot
-    # nrfx is used to ensure that variables are stored in RAM, so no qemu.
-    platform_exclude: qemu_cortex_m3 qemu_x86
+    # nrfx is used to ensure that variables are stored in RAM, so no qemu
+    # or native posix
+    platform_exclude: qemu_cortex_m3 qemu_x86 native_posix
```

Comment:
Which platforms will this run on now?

Reply:
all nrf platforms, when CI starts supporting that, if it already does not :)

Reply:
It does not 😄 
And I'm not sure it will, actually. In upstream, all twister tests run on emulators or native_posix, as they all run on build servers without specific target hardware. I assume we want to make the same distinction here.

The CI run that passed for this PR now skipped this test in all configurations, as it only builds for qemu_cortex_m3 and native_posix. There are three options to make this test do anything:
1. Make it work for one of the two platforms twister runs tests on
2. Convert it to a system test and push it to one of the test-blabla repos
3. Add support for a new platform in CI, where this test is supported, such as babbleSim.

Reply:
@PerMac is actually working on adding hw targets for twister. I'll approve so we can get this in, but this test should probably be taken into consideration in that work